src/channeldef.ts

```diff
@@ -30,13 +31,15 @@ import {getFullName, QUANTITATIVE, StandardType, Type} from './type';
 import {contains, flatAccessWithDatum, getFirstDefined, internalField, replacePathInField, titlecase} from './util';
 
 export type Value = number | string | boolean | null;
+export type Gradient = GradientLinear | GradientRadial;
```

Comment:
`GradientLinear` and `GradientRadial` will become class names in Altair.  I think Linear/RadialGradient sound more like a proper class name?  (Yes, I think we should fix this in Vega too.)

Reply:
It's actually nice to be able to type "gradient" and then see the options in autocomplete, though, no?

Reply:
That’s true but it’s reading weird. Most other APIs use linear gradient.

Reply:
I'm renaming it but we also need to rename the Vega schema to make it consistent.